src/foam/dashboard/view/DAOTable.js

```diff
@@ -171,12 +180,17 @@ foam.CLASS({
       name: 'viewMoreAction',
       label: 'View more activities',
       code: function() {
-        this.stack.push(this.StackBlock.create({
-          view: {
-            class: this.DAOBrowseControllerView,
-            data: this.dao.where(this.predicate),
-          }, parent: this.__subContext__
-        }));
+        if ( ! (this.viewMoreMenuItem === undefined ||
+                this.viewMoreMenuItem === '') ) {
+          this.pushMenu(this.viewMoreMenuItem);
+        } else {
+          this.stack.push(this.StackBlock.create({
+            view: {
+              class: this.DAOBrowseControllerView,
+              data: this.dao.where(this.predicate),
+            }, parent: this.__subContext__
```

Comment:
Can you just use 'this' here?

Reply:
the original code did this. i left it alone